pkg/utils/log/log.go

```diff
@@ -68,30 +67,30 @@ func setLogger(v *int, debug *bool) {
 		encoderConf := zap.NewDevelopmentEncoderConfig()
 		encoderConf.EncodeLevel = zapcore.CapitalColorLevelEncoder
 		encoder = zapcore.NewConsoleEncoder(encoderConf)
-
-		opts = append(opts, zap.Development(), zap.AddStacktrace(zap.ErrorLevel))
 	} else {
 		encoderConf := zap.NewProductionEncoderConfig()
 		encoderConf.MessageKey = "message"
 		encoderConf.TimeKey = "@timestamp"
 		encoderConf.EncodeTime = zapcore.ISO8601TimeEncoder
 		encoder = zapcore.NewJSONEncoder(encoderConf)
 
-		opts = append(opts, zap.AddStacktrace(zap.ErrorLevel))
 		if zapLevel.Level() > zap.DebugLevel {
```

Comment:
This block is not necessary because the controller-runtime handles it.

Reply:
removed!